runtime/ftplugin/vim.vim

```diff
@@ -48,7 +48,22 @@ setlocal fo-=t fo+=croql
 setlocal isk+=#
 
 " Use :help to lookup the keyword under the cursor with K.
-setlocal keywordprg=:help
+" Distinguish between commands, options and functions.
+function! s:Help(args)
+  silent! let syn_name = synIDattr(synID(line('.'), col('.'), 1), 'name')
+
+  if syn_name =~# 'vimCommand'
+    execute 'help' a:args..':'
```

Comment:
```suggestion
    execute 'help :'..a:args
```

Reply:
Thank you @AndrewRadev . Now it show's that I just tested the distinction between functions and variables.